### PR TITLE
answers display creation

### PR DESCRIPTION
**Pull request by Gonaline**

### CODE REVIEW

**Review by n-faycel — DISMISSED** [review]
ok for me!

File: frontend/package.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
     "react-dom": "^17.0.2",
     "react-router-dom": "^6.3.0",
     "styled-components": "^5.3.5",
-    "styled-reset": "^4.3.4"
+    "styled-reset": "^4.3.4",
+    "template-fullstack": "file:.."
```
Comment (LuigiMenez) [inline frontend/package.json]: Nop

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Not template fullstack in package.json

**Review by n-faycel — APPROVED** [review]
ok for me!

**Review by LuigiMenez — APPROVED** [review]
Ok for me

File: backend/package.json [hunk 2]
```diff
@@ -17,7 +17,8 @@
     "cors": "^2.8.5",
     "dotenv": "^10.0.0",
     "express": "^4.17.2",
-    "mysql2": "^2.3.3"
+    "mysql2": "^2.3.3",
+    "template-fullstack": "file:.."
```
Comment (LoicBrassart) [inline backend/package.json]: Nope.

File: frontend/src/components/Buttons/Answer/index.jsx [hunk 3]
```diff
@@ -1,22 +1,25 @@
 import PropTypes from "prop-types";
 import Style from "./style";
 
-export default function Answer({ data: { correctAnswer } }) {
+export default function Answer({ data: { correctAnswer, incorrectAnswers } }) {
+  const answers = [];
```
Comment (LoicBrassart) [inline frontend/src/components/Buttons/Answer/index.jsx]: Plz use a state instead

const  [answers, setAnswers] = useState([])
setAnswers(answers.sort(() => Math.random() - 0.5);)

Comment (LoicBrassart) [reply frontend/src/components/Buttons/Answer/index.jsx]: const tmpAnswers = [...answers]

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Use a state (this case is complicated, I'll help)

**Comment by LoicBrassart** [comment]
- Fix conflicts
- Do you need help about this ? (answer on Slack or in person plz)

**Comment by LoicBrassart** [comment]
Plz fix conflicts before assigning

**Pull request merged**